SWLOR.Game.Server/Native/ResolveAttackRoll.cs

```diff
@@ -271,19 +299,72 @@ private static void OnResolveAttackRoll(void* thisPtr, void* pTarget)
             Log.Write(LogGroup.Attack, $"attackerAccuracy = {attackerAccuracy}, modifiers = {accuracyModifiers}, defenderEvasion = {defenderEvasion}");
             Log.Write(LogGroup.Attack, $"Hit Rate: {hitRate}, Roll = {attackRoll}");
 
+            var defenderWeapon = defender.m_pInventory.GetItemInSlot((uint)EquipmentSlot.RightHand);
+            var defenderOffhand = defender.m_pInventory.GetItemInSlot((uint)EquipmentSlot.LeftHand);
+            bool saberBlock = false;
+
+            if(defenderWeapon != null && (
+                (BaseItem)defenderWeapon.m_nBaseItem == BaseItem.Lightsaber ||
+                (BaseItem)defenderWeapon.m_nBaseItem == BaseItem.Saberstaff))
+            {
+                for(var i = 0; i < defenderWeapon.m_lstPassiveProperties.Count; i++)
+                {
+                    var ip = defenderWeapon.GetPassiveProperty(i);
+                    if (ip != null && ip.m_nPropertyName == (ushort)Core.NWScript.Enum.Item.ItemPropertyType.Light)
```

Comment:
Let's add Core.NWScript.Enum.Item as a using statement to simplify this line and make it more readable.